### ISSUE TITLE
fail build master :WARNING: unrecognized options: --enable-tirpc

### ISSUE THREAD

**Issue by inevity** [gluster/gluster-block#212]
Centos 7.6
if --enable-tirpc = no ,omit error as titel.
if ----enable-libtirpc= no, build with tirpc.

**Comment by pkalever** [comment]
Hello @inevity,

Hopefully you are running the latest master code.

I can see that the latest build for master with travis CI succeeds on centos7 newer docker image with kernel-3.10.0-957.10.1.el7.x86_64

Details: https://travis-ci.com/gluster/gluster-block/builds/108668129#L2575

Did you run autogen.sh after pulling the latest code by the way ?

\# make distclean
\#./autogen.sh  && ./configure --enable-tirpc=no

**Comment by pkalever** [comment]
@inevity Just in case, if you are giving space before and after '=' with flag '--enable-tirpc', then you should look for that :-)

This is wrong:
```
# ./configure --enable-tirpc = no
```

You should change it to:
```
# ./configure --enable-tirpc=no
```

**Comment by inevity** [comment]
Look at line 2773@pkalever

**Comment by pkalever** [comment]
@inevity thanks for reporting the issue, should be fixed with #214
